{"uuid": "6d7f1991-3ba9-4f80-9da3-c2be788a2dce", "children": ["85320108-0ca1-4983-82f8-e78e618ce6db"], "befores": [{"name": "login_user", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"button_button__33qZ0 button_button_type_primary__1O7Bx button_button_size_medium__3zxIa\"> is not clickable at point (750,549) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:197:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:342:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:344:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:220:31\n\n", "trace": "  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/tests/conftest.py\", line 25, in login_user\n    login(driver, user)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/utils/user_api.py\", line 35, in login\n    driver.find_element(\"xpath\", \"//button[text()='Войти']\").click()\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py\", line 119, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py\", line 572, in _execute\n    return self._parent.execute(command, params)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/yulianna_lescheva/PycharmProjects/Diplom_3/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1745184131438, "stop": 1745184132228}], "start": 1745184131438, "stop": 1745184132242}